paddle/fluid/operators/multiplex_op.cu

```diff
@@ -29,6 +29,10 @@ class MultiplexGPUKernel : public framework::OpKernel<T> {
     auto* out = ctx.Output<Tensor>("Out");
     out->mutable_data<T>(ctx.GetPlace());
 
+    PADDLE_ENFORCE_GT(ins[0]->numel(), 0,
```

Comment:
Why only check the first tensor of the input? Shouldn't it be checked more than one becasue of MultiInput type?

Reply:
Done